drivers/google/google_test.go

```diff
@@ -0,0 +1,153 @@
+package google
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/codegangsta/cli"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDriverName(t *testing.T) {
+	driverName := newDriver().DriverName()
+
+	assert.Equal(t, "google", driverName)
+}
+
+func TestFailWithoutProject(t *testing.T) {
+	_, err := create([]string{"test-name"})
+
+	assert.EqualError(t, err, "Please specify the Google Cloud Project name using the option --google-project.")
+}
+
+func TestParseFlags(t *testing.T) {
+	driver, err := create([]string{
```

Comment:
You are right, the parsing is ok but unless we test at his level, we can't see the problem. The test will inject a `StringSlice` and the code will read a `StringSlice`, it will be green and it will fail at runtime because there's somewhere else a flag definition that is not aligned with `SetConfigFromFlags`. See what I mean?